src/timeline.ts

```diff
@@ -305,17 +337,87 @@ class Timeline implements IAppView {
                   open2dHistogram = this.parentNode;
                 }
               });
+           }
 
-          });
-      });
-    }
+           //all changes
+           drawBars(data, color);
+          //helper variable for clicking remove button!
+           var btnClicked = 0;
```

Comment:
Use a boolean instead of number and rename to `isBtnClicked` (or something similar) to clarify it's a status variable.